README.md

```diff
@@ -2,7 +2,8 @@
 
 [MBK keycap profile](https://fkcaps.com/keycaps/mbk) has been designed by **mburger** and is being manufactured by [FKcaps](https://fkcaps.com). They are low profile keycaps, meant to fit Kailh Choc switches.
 
-> These models are provided for ***rendering purpose only*** and are not licensed for any commercial use.
+> These models are provided for ***rendering purpose only*** and are not licensed for any other commercial use than renderings.
```

Comment:
what do you think of 
> These models are provided for ***rendering purpose only*** and are not licensed for any commercial or manufacturing use (rendering for a commercial product is allowed).

Reply:
Updated wording to better capture  what you mentioned. But also trying to keep it not confusing.